keps/3122-expose-flavors-in-localqueue-status/kep.yaml

```diff
@@ -0,0 +1,28 @@
+title: Expose Flavors in LocalQueue Status
+kep-number: 3122
+authors:
+  - "@mbobrovskyi"
+status: implementable
+creation-date: 2024-10-02
+reviewers:
+  - "@mimowo"
+  - "@alculquicondor"
+  - "@tenzen-y"
+approvers:
+  - "@mimowo"
+  - "@alculquicondor"
+  - "@tenzen-y"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
```

Comment:
Will we introduce the feature gate?
If not, this may be able to say stable.

cc: @mimowo

Reply:
Good point. This feature is relatively simple so I wouldn't mind going to `stable`. 

OTOH it might be a better practice to support disabling, especially for non opt-in features, so I'm leaning slightly towards introducing `ExposeFlavorsInLocalQueue`, enabled by default. Then we also say `disable-supported: true`. It will also leave us the KEP for future extensions list `availableCoveredResources`.

Reply:
> OTOH it might be a better practice to support disabling, especially for non opt-in features, so I'm leaning slightly towards keeping beta introducing ExposeFlavorsInLocalQueue, enabled by default. Then we also say disable-supported: true. It will also leave us the KEP for future extensions list availableCoveredResources.

That sounds reasonable to me. Let's keep staying the beta and adding the featureGate during the v0.9.0.
@mbobrovskyi Could you add a featureGate?

Reply:
Done